cli/cmd/project/delete.go

```diff
@@ -25,24 +25,32 @@ func DeleteCmd(cfg *config.Config) *cobra.Command {
 			}
 			defer client.Close()
 
-			if !cmd.Flags().Changed("project") {
+			if len(args) > 0 {
+				name = args[0]
+			}
+
+			if !cmd.Flags().Changed("project") && len(args) == 0 && cfg.Interactive {
 				name, err = inferProjectName(cmd.Context(), client, cfg.Org, path)
 				if err != nil {
 					return err
 				}
 			}
 
+			if name == "" {
+				return fmt.Errorf("please provide valid project name, Run `rill project list` for available projects")
```

Comment:
Fix grammar: `"Please provide a valid project name. Run `rill project list` to see the available projects."`

Reply:
making it as `please provide a valid project name. Run \`rill project list\` to see the available projects`